jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -59,6 +61,7 @@ def load_profiles(self, secret_name="jupyter-singleuser-profiles", filename=None
         if config_map_yaml:
           self.sizes.extend(config_map_yaml.get("sizes", [self.empty_profile()]))
           self.profiles.extend(config_map_yaml.get("profiles", [self.empty_profile()]))
+          self.ui = config_map_yaml.get("ui", [self.empty_profile()])
```

Comment:
I don't think that putting empty_profile there by default makes sense - that would probably result in failed validation, right?